WordPress/Classes/ViewRelated/Blog/Site Picker/BlogList/BlogListViewModel.swift

```diff
@@ -0,0 +1,107 @@
+import SwiftUI
+
+final class BlogListViewModel: ObservableObject {
```

Comment:
Have you considered using `NSFetchedResultsController` instead of manually keeping track of the blogs in variables?

We can setup one or multiple `NSFetchedResultsControllers` for each section and bind there content to the view. Then updating the blog in core data would directly update the view. This could simplify this class quite a bit. What do you think?

Reply:
Won't we need the variables to bind to the view in either case? A `@Published` var will be necessary for each section if I am not missing something.

Reply:
Yes. Using `NSFetchedResultsControllers` will simplify the following:
- No need to keep a separate copy of all blogs in memory.
- No need to manually filter and sort each section when a change happens.
- No need to worry about how future changes to the code could trigger a recalculation of a section.

Reply:
Resolved in https://github.com/wordpress-mobile/WordPress-iOS/pull/22833/commits/ad5f5c26cb86249c900e9ccc9ef8e6f789f3655d